nsxt/resource_nsxt_policy_edge_cluster.go

```diff
@@ -376,7 +376,7 @@ func resourceNsxtPolicyEdgeClusterDelete(d *schema.ResourceData, m interface{})
 
 	connector := getPolicyConnector(m)
 	client := enforcement_points.NewEdgeClustersClient(connector)
-	err = client.Delete(siteID, epID, id)
+	err = client.Delete(siteID, epID, id, nil)
```

Comment:
Why are you making this change? Is it due to SDK update? (If so, what should the 4th argument be)

Reply:
This one is derived from PR #1610, and its:

deleteMemberEdgeNodesParam Flag to specify whether to delete edge transport nodes within edge cluster. (optional, default to false)